plugin/src/main/groovy/com/facebook/testing/screenshot/build/ScreenshotsPlugin.groovy

```diff
@@ -119,12 +130,16 @@ class ScreenshotsPlugin implements Plugin<Project> {
     }
 
     project.task("recordMode") {
+      group = project.screenshots.GROUP
+      description = "Run all the screenshot tests and record all the screenshots in your screenshots folder"
```

Comment:
Lets use `"Run all screenshot tests and record all screenshots in your screenshots folder"`